### PR TITLE
Cover `DomEvent` with unit tests

### LINKED ISSUE #7434: DomEvent functions not covered by tests
We have some [DomEvent](https://github.com/Leaflet/Leaflet/blob/master/src/dom/DomEvent.js) tests here: https://github.com/Leaflet/Leaflet/blob/master/spec/suites/dom/DomEventSpec.js, but not all functions are covered.

Here the list of lacking tests:
1. `on(<HTMLElement> el, <String> types, <Function> fn, <Object> context?)` (where `types` has several events)
2. `on(<HTMLElement> el, <Object> eventMap, <Object> context?)`
3. `off(<HTMLElement> el, <String> types, <Function> fn, <Object> context?)` (where `types` has several events)
4. `off(<HTMLElement> el, <Object> eventMap, <Object> context?)`

Note: some `on`/`off` tests implemented in #7125.

5. `stop(<DOMEvent> ev)`
6. `getMousePosition(<DOMEvent> ev, <HTMLElement> container?)`
7. `getWheelDelta(<DOMEvent> ev)`

Ref: https://leafletjs.com/reference-1.7.1.html#domevent

Note:
When implementing required test cases consider minor refactoring in #7438 (which already has tests for `disableScrollPropagation` and `disableClickPropagation`).

---
Related:
- [x] `DomEvent.DoubleTap.js` is covered by #7027.
- [x] `DomEvent.Pointer.js` is covered by #7415.

### PR DESCRIPTION
**Pull request by stephenspol**
Closes #7434. Not sure everything was covered. Feedback is welcomed.

### CODE REVIEW

File: spec/suites/dom/DomEventSpec.js [hunk 1]
```diff
@@ -101,7 +343,7 @@ describe('DomEvent', function () {
 		});
 	});
 
-	describe('#off (removeListener)', function () {
+	describe.only('#off (removeListener)', function () {
```
Comment (mourner) [inline spec/suites/dom/DomEventSpec.js:346]: Looks like a leftover, right? 

```suggestion
	describe('#off (removeListener)', function () {

```

**Review by mourner — CHANGES_REQUESTED** [review]
This is great, thanks so much for taking on this!

File: spec/suites/dom/DomEventSpec.js (hunk 1 above)
Comment (stephenspol) [reply spec/suites/dom/DomEventSpec.js:346]: Yep

**Comment by Falke-Design** [comment]
Does this make #7453 obsolete?

**Comment by Falke-Design** [comment]
@stephenspol please run `npm run lintfix`

**Review by mourner — APPROVED** [review]
Great! Can you also please check if #7453 is still relevant?

**Comment by Falke-Design** [comment]
I closed #7453 no need to keep it open.

**Pull request merged**